src/App.js

```diff
@@ -37,9 +38,13 @@ const tweetsArray = [
 function App() {
   return (
     <div className="App">
-      <Tweet />
+      <Tweet tweet={tweetsArray[0]}></Tweet>
```

Comment:
And here, you could've mapped the tweetsArray, like:

`{tweetsArray.map((el) =>{
      return <Tweet tweet={el}/>  
    })}`